### DISCUSSION TITLE
Commit Directory

### DISCUSSION THREAD

**Discussion by Bhavneet345** [Lakhankumawat/LearnCPP#462] [Q&A] [answered]
I have raised an issue named Diameter of a Binary Tree, can you please tell me where I can commit, or do I need to create a new folder and then commit??

**Comment by Lakhankumawat** [comment, marked as answer]
Please mention your ISSUE number

**Reply by Bhavneet345** [reply]
353

**Reply by Lakhankumawat** [reply]
LearnCPP/S-Stack/LinkedListStack.cpp and README.md changes will be here

**Reply by Bhavneet345** [reply]
The binary tree code has to merged in stack folder??
